aws-lc-fips-sys/builder/cmake_builder.rs

```diff
@@ -85,9 +90,18 @@ impl CmakeBuilder {
             } else {
                 cmake_cfg.define("CMAKE_BUILD_TYPE", "release");
             }
-        } else if target_os() != "windows" {
+        } else if target_os() == "windows" {
             // The Windows/FIPS build rejects "debug" profile
             // https://github.com/aws/aws-lc/blob/main/CMakeLists.txt#L656
+            cmake_cfg.define("CMAKE_BUILD_TYPE", "relwithdebinfo");
+            cmake_cfg.generator("Ninja");
```

Comment:
Is Ninja explicitly required on Windows? Does the MSVC generator not work?

Reply:
Unfortunately so. I've not been able to get the MSVC generator working. The binary the MSVC generator produces for the FIPS build doesn't pass the bcm "self test".